### PR TITLE
[Task #67928, #67927] Tạo dashboard

### PR DESCRIPTION
**Pull request by vutuankiet4599**
## Related Tickets:

-   [Task #67927](https://edu-redmine.sun-asterisk.vn/issues/67927)
-   [Task #67928](https://edu-redmine.sun-asterisk.vn/issues/67928)

## WHAT this PR does?

- Tạo controller và giao diện cho admin dashboard

## Notes Impacted Areas:

-   N/A.

## Attachments:

Screenshot 2023-09-17 122945

### CODE REVIEW

File: resources/views/dashboard.blade.php [hunk 1]
```diff
@@ -1,17 +1,44 @@
...
+                <p class="text-xl">{{ __("Total films") }}</p>
+                <p class="text-4xl">{{ $totalFilms }}</p>
+            </div>
+            <div class="w-64 h-36 bg-indigo-500 text-white p-5 flex flex-col justify-around rounded-md">
+                <p class="text-xl">{{ __("Total tickets") }}</p>
+                <p class="text-4xl">{{ $totalTickets }}</p>
+            </div>
+            <div class="w-64 h-36 bg-fuchsia-500 text-white p-5 flex flex-col justify-around rounded-md">
+                <p class="text-xl">{{ __("Sum price") }}</p>
+                <p class="text-4xl">{{ $sumPrice }}</p>
```
Comment (namnt0413) [inline resources/views/dashboard.blade.php]: <p class="text-4xl">{{ number_format($sumPrice) }}</p>

**Review by namnt0413 — CHANGES_REQUESTED** [review]

**Comment by namnt0413** [comment]
ok

**Comment by vutuankiet4599** [comment]
ready

File: app/Http/Controllers/DashboardController.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    {
+        $totalFilms = DB::table('films')->count('id');
+        $totalTickets = DB::table('tickets')->count('id');
+        $sumPrice = DB::table('tickets')->sum('price');
+        $mostPopularFilm = Ticket::groupBy('film_id')
+                            ->selectRaw('film_id, sum(price) as sum_price')
+                            ->orderBy('sum_price', 'desc')->first()->film;
+        $mostBookingUser = Ticket::groupBy('user_id')
+                            ->selectRaw('user_id, count(film_id) as count_film')
+                            ->orderBy('count_film', 'desc')->first()->user;
```
Comment (ghost) [inline app/Http/Controllers/DashboardController.php]: indent 4 spaces

File: resources/views/dashboard.blade.php [hunk 3]
```diff
@@ -1,17 +1,44 @@
...
-        <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">
-            <div class="bg-white overflow-hidden shadow-sm sm:rounded-lg">
-                <div class="p-6 bg-white border-b border-gray-200">
-                    You're logged in!
-                </div>
+    <!-- <div>{{ $totalFilms }}</div>
+    <div>{{ $totalTickets }}</div>
+    <div>{{ $sumPrice }}</div>
+    <div>{{ $mostPopularFilm->title }}</div>
+    <div>{{ $mostBookingUser->username }}</div> -->
```
Comment (ghost) [inline resources/views/dashboard.blade.php]: ko dùng xóa đi chứ

File: resources/views/dashboard.blade.php (hunk 1 above)
Comment (ghost) [reply resources/views/dashboard.blade.php]: team member commented

**Comment by vutuankiet4599** [comment]
ready

File: app/Http/Controllers/DashboardController.php (hunk 2 above)
Comment (ghost) [reply app/Http/Controllers/DashboardController.php]: ??? not fix??

File: resources/views/dashboard.blade.php (hunk 3 above)
Comment (ghost) [reply resources/views/dashboard.blade.php]: not fixed also

**Comment by vutuankiet4599** [comment]
ready

**Pull request merged**